app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  before_action :set_commentable, only: %i[create destroy]
```

Comment:
```suggestion
  before_action :set_commentable
```

今のところ全アクションが対象になるのでonlyは不要かもしれません